builder/Dockerfile

```diff
@@ -35,7 +35,9 @@ RUN apk --no-cache add alpine-sdk \
                        cmake \
                        musl-dev \
                        postgresql-dev \
-                       mongo-c-driver-dev
+                       mongo-c-driver-dev \
+                       protobuf \
+                       snappy
```

Comment:
These two will need to be the `-dev` packages, otherwise the builder image will be missing the header files that are needed to build and link against the libraries.

Reply:
Done :)